groupdir/create_group_swdir.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+if [ -z $1 ]; then
+   echo "Supply the name of the group"
+   echo "and optionally the quota in GiB and again optionally the inode limit"
+   exit
+fi
+
+groupname=$1
+gid=$( getent group $groupname | awk -F ':' '{print $3}' )
+if [ -z $gid ]; then
+   echo "Group $groupname cannot be found"
+   exit
+fi
+
+swdir=/userapps
+echo $swdir
+
+sudo mkdir $swdir/$groupname
+sudo chown root.$1 $swdir/$groupname
+sudo chmod 0770 $swdir/$groupname
+sudo chmod g+s $swdir/$groupname
+#
+sudo setfacl -d -m group:$groupname:rwX $swdir/$groupname
+# 
+# Set quota on software directory
+#
+ssh 172.23.15.207 sudo xfs_quota -x -c \'project -s -p /nfs/userapps/${groupname} ${gid}\' /nfs
+
+if [ -z $2 ]; then 
+   echo "Set space quota to defaults"
+   ssh 172.23.15.207 sudo xfs_quota -x -c \'limit -p bsoft=0 bhard=0 ${gid}\' /nfs
```

Comment:
I'm not completely sure how this works, the default limit here would be 0 bytes (i.e., no data), right?

Reply:
0 means using the default. This is a bit confusing, but I wanted to make sure that it is possible to revert to the defaults, instead of a fixed value that corresponds to the default.